lib/flat_search/olx_scraper.ex

```diff
@@ -142,10 +153,20 @@ defmodule FlatSearch.OlxScraper do
     end
   end
 
-  defp get_nested_element(enum) do
+  defp get_nested_element(enum) when enum != [] do
     enum
     |> Enum.at(0)
     |> elem(2)
     |> Enum.at(0)
   end
```

Comment:
this would error if enum == [] 😛 
add another definition below, to catch this case